gateway-server/src/main/java/org/apache/knox/gateway/websockets/ProxyWebSocketAdapter.java

```diff
@@ -87,6 +87,8 @@ public void onWebSocketConnect(final Session frontEndSession) {
      * plumbing takes place
      */
     container = ContainerProvider.getWebSocketContainer();
+    container.setDefaultMaxBinaryMessageBufferSize(Integer.MAX_VALUE);
+    container.setDefaultMaxTextMessageBufferSize(Integer.MAX_VALUE);
```

Comment:
Would be great to use the configs for this instead of hard coding to `Integer.MAX_VALUE`. I haven't looked to see how easy it is to access the gateway config here.

Reply:
I can grab those from the frontEndSession. Originally I was trying to figure out how to just set the policy on the container but as far as I can tell Jetty doesn't provide a good way to do that. The backendSession is still use Integer.MAX_VALUE from another effort. Should I try to set it as well?

Reply:
The backendSession has hardcoded Integer.MAX_VALUE unfortunately which needs to be fixed as well, feel free to open a bug and I will fix this. It appears that this value has no effect.